config/test_settings.py

```diff
@@ -0,0 +1,34 @@
+# Start with the regular settings
+
+# Speeds up test execution.
+# See https://brobin.me/blog/2016/08/7-ways-to-speed-up-your-django-test-suite/
+
+# noinspection PyUnresolvedReferences
+from .settings import *  # noqa: F401,F403
```

Comment:
Just saw this. Well then it should theoretically work, as it also loads `local_settings`.